common/models/create_service_request.go

```diff
@@ -11,6 +11,7 @@ type CreateServiceRequest struct {
 	EnvironmentID  string `json:"environment_id"`
 	LoadBalancerID string `json:"load_balancer_id"`
 	ServiceName    string `json:"service_name"`
+	Scale          int    `json:"scale"`
```

Comment:
[rambling] I think I see why you did this now... easier to define the scale upfront for use with terraform. I do see some edge cases, however, with systems like Consul where you still need to set scale later on after the first service is officially up.

Reply:
yeah this doesn't remove that behavior, the default scale is still 1